### PR TITLE
minor/PLAT-866/add-cos-v2

### PR DESCRIPTION
**Pull request by zhelinisivanesan**

### CODE REVIEW

**Comment by zhelinisivanesan** [comment]
Linting will be fixed merging #44 first

File: satellitevu/apis/orders.py [hunk 1]
```diff
@@ -177,3 +192,202 @@ def _save_order_to_zip(self, destdir: str, order_id: UUID, item_ids: List[str]):
...
+    ) -> Dict:
+        """
+        Retrieve details of an imagery order.
+
+        Args:
+            contract_id: String or UUID representing the ID of the Contract
+            which an order is associated with.
+
+            order_id: UUID representing the order id e.g.
+            "2009466e-cccc-4712-a489-b09aeb772296".
```
Comment (jharrisonSV) [inline satellitevu/apis/orders.py:236]:
```suggestion
            order_id: String or UUID representing the order id e.g.
            "2009466e-cccc-4712-a489-b09aeb772296".
```

Comment (jharrisonSV) [reply satellitevu/apis/orders.py:236]: Same applies for other v2 methods with `order_id` as a parameter

File: satellitevu/apis/orders.py [hunk 2]
```diff
@@ -177,3 +192,202 @@ def _save_order_to_zip(self, destdir: str, order_id: UUID, item_ids: List[str]):
...
+        return self._save_order_to_zip(destdir, contract_id, order_id, item_ids)
+
+    def _save_order_to_zip(
+        self,
+        destdir: str,
+        contract_id: Union[UUID, str],
+        order_id: UUID,
+        item_ids: List[str],
+    ):
+        destzip = os.path.join(destdir, f"SatelliteVu_{order_id}")
```
Comment (jharrisonSV) [inline satellitevu/apis/orders.py:381]: Do we want to change the destination directory to `SatVu` for v2?

**Review by jharrisonSV — APPROVED** [review]

File: satellitevu/apis/orders.py (hunk 2 above)
Comment (zhelinisivanesan) [reply satellitevu/apis/orders.py]: yeah good shout

**Pull request merged**